### DISCUSSION TITLE
I need to change color of Green Line appears at the time of Drag & Drop Component

### DISCUSSION THREAD

**Discussion by pardeepjainsolulab** [GrapesJS/grapesjs#5598] [Q&A]
Hello community, I need to change color of this Green Line appears at the time of Drag & Drop Component, as shown in the picture.

I have tried multiple ways to inspect it out but nothing works till now.
Is this image or css built line? also is there any predefined class for the same?

Any help would be appreciated.
 
<img width="664" alt="Screenshot 2024-01-06 at 4 33 52 PM" src="https://github.com/GrapesJS/grapesjs/assets/138665951/e013586e-d7d8-4984-bad2-584770574411">

<img width="1470" alt="Screenshot 2024-01-06 at 4 36 05 PM" src="https://github.com/GrapesJS/grapesjs/assets/138665951/1ce2d78b-dec7-44dc-b01f-2954cff9c38f">

**Comment by abhishekSolulab** [comment]
.gjs-placeholder.horizontal, .gjs-com-placeholder.horizontal, .gjs-placeholder.horizontal {
  border-color: transparent #363636b0 !important;
    // border-width: 3px 5px;
    // margin: -3px 0 0;
}
.gjs-placeholder-int, .gjs-com-placeholder-int, .gjs-placeholder-int{
  background-color: #363636b0 !important;
}

please override css with custom css.
